### ISSUE TITLE
push to pip?

### ISSUE THREAD

**Issue by jayhesselberth** [nyergler/hieroglyph#82]
Hi Nathan – would you mind pushing the latest version (0.7) to pip? The current pip version (0.6.5) doesn't have the `slides2` theme.

Thanks

**Comment by nyergler** [comment]
There are a few bugs open that I think need to be fixed before doing a "real" 0.7 release. You should be able to get the latest from Github by doing:

```
$ pip install hieroglyph==dev
```

If you're on a recent version of pip, you'll also need to tell it to allow installation of pre-release versions and externally hosted distributions; something like:

```
$ pip install --pre --allow-external hieroglyph hieroglyph==dev 
```

Will that work for you? I can also cut an 0.7 pre-release (probably tagged as a beta or something) and push that to PyPI if that'd be better (you may still need to specify `--pre` even then, I'm not sure).

**Comment by nyergler** [comment]
I just cut the 0.7 release, and pushed the new version to PyPI/pip.
